pywbem_mock/_wbemconnection_mock.py

```diff
@@ -543,20 +546,102 @@ def compile_mof_string(self, mof_str, namespace=None, search_paths=None,
 
         mofcomp.compile_string(mof_str, namespace)
 
+    def compile_schema_classes(self, schema_mof_pragma_file, schema_classnames,
+                               namespace=None, verbose=False):
```

Comment:
1. I suggest to switch the order of parameters `schema_mof_pragma_file` and `schema_classnames` to be consistent with the global function `build_schema_mof()`, and because the class names are sort of the main input.
2. Please rename the `schema_classnames` parameter to `class_names`, for consistency.